documentation/assertions/Promise/to-be-fulfilled.md

```diff
@@ -46,13 +44,13 @@ a regular expression, a function, or an object:
 
 
 ```javascript#async:true
```

Comment:
It's still needed by unexpected-site-generator, but we should be able to use a similar technique to avoid it there.